style.css

```diff
@@ -137,3 +137,430 @@ ul {
   top: 430px;
   left: 24px;
 }
+
+.description {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 120px 24px;
+  gap: 12px;
+  position: absolute;
+  width: 327px;
+  height: 200px;
+  left: 0;
+  top: 1330px;
+}
+
+.multi {
+  width: 327px;
+  height: 36px;
+  font-family: 'Crete Round', serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+  display: flex;
+  align-items: center;
+  color: #091e42;
+}
+
+.daily {
+  width: 327px;
+  height: 147px;
+  font-family: Inter, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #091e42;
+}
+
+.languages {
+  list-style: none;
+  position: absolute;
+  top: 300px;
+  width: 297px;
+  height: 12px;
+  display: grid;
+  grid-template-columns: auto auto auto auto;
+  padding: 10px;
+  gap: 5px;
+}
+
+.g-item {
+  box-sizing: border-box;
+  border: 1px solid #8993a4;
+  padding: 10px 12px;
+  background: #fff;
+}
+
+div#frams {
+  position: absolute;
+  top: 1800px;
+  width: 340px;
+  height: 2610px;
+  display: grid;
+  grid-template-columns: auto;
+  padding: 10px;
+}
+
+.grid-item {
+  background-image: url(./images/Img\ Placeholders.png);
+  font-size: 30;
+  height: auto;
+  border: none;
+  position: relative;
+}
+
+.but {
+  position: absolute;
+  bottom: 0;
+  width: 340px;
+  height: 48px;
+  background: #ff6b00;
+  font-family: inter, sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  border: none;
+  color: #fff;
+}
+
+.but:hover {
```

Comment:
:red_circle: This is not the right hover state. Please kindly Implement the button interactions (enable, hover, etc.). according to the project requirements. You can select the hover state from the Figma Design itself as follow :arrow_down: 
![figma](https://user-images.githubusercontent.com/26736582/212466779-9dfe49fd-52e4-4792-8e38-191ba50cdc04.gif)